### ISSUE TITLE
"Concert Pitch" is an ambiguous term and above that it's also symbolized in an misleading way

### ISSUE THREAD

**Issue by oMrSmith** [musescore/MuseScore#16746]
### Your idea

Please consider this request more as a feedback.
I don't think it's gonna be open very long.
Still I'll try to be as clear and short as possible.

I believe:

1. "Concert Pitch" should better be called "Sounding Pitch".
2. Neither for "Concert Pitch" nor for "Sounding Pitch" should the symbol be a tuning fork.

### Problem to be solved

Allow me to explain why I think so:

1. "Concert Pitch" is an ambiguous term which is also used for tuning.
In Germany for example "Concert Pitch" is only associated with that meaning.

![image](https://user-images.githubusercontent.com/78870770/224510020-4c12ef9b-1253-47ce-a428-345b12f6af6f.png)

I myself still cannot remember weather "Concert Pitch" actually means "written" or "sounding" pitch,
since the term does not contain this information. I find myself toggling the button to find it out.

2. The tuning fork symbol makes it seem as though 
![image](https://user-images.githubusercontent.com/78870770/224510205-dc187fad-7f21-49e8-b86e-34920cb8d775.png)
"Concert Pitch" would have to do with the frequency of the master-tuning, 
which in the giving case it has not.

### Additional context

Final thoughts:

"Sounding Pitch" would be a clear term 
which contains the actual meaning inside its name.

Because of that it would need no symbol to explain itself
so that both problems could be solved at once.

**Comment by cbjeukendrup** [comment]
I believe that the word "concert pitch" is a sort of industry standard in music notation software, so I think that will stay. However, the tuning fork icon might be misleading indeed, given that "concert pitch" has a secondary meaning. (I did not know that by the way, to be honest).

Just another idea: instead of a checkbox, we could make it a dropdown box, with options "Written pitch" and "Concert pitch". I think that would make it completely unambiguous (except to people who are completely unfamiliar with the whole concept of transposing instruments, of course). But the big disadvantage of a dropdown is that it requires two clicks instead of one, and it might look a bit more cluttered, so I'm afraid the designers won't like it much.

**Comment by oMrSmith** [comment]
I agree: A dropdown menu would slow down the workflow, maybe something like that:
![image](https://user-images.githubusercontent.com/78870770/224514000-e006e240-e08c-4aa6-8d52-39ff89c527fa.png)

**Comment by MarcSabatella** [comment]
To me as a US-based native English speaker, the term concert pitch is absolutely the standar, not just in notation software but in music in general.  *However*, I also agree the tuning fork icon is not helpful.  And I agree that the current checkbox is somehow not as clear as it could be.  I'm constantly looking at it and seeing the word "concert pitch" (not noticing the checkbox state) and assuming it means the score is actually being displayed in concert pitch.  After all, when it says "Page view", it *means* it - you don't also have to look at a checkbox to see if it's on or off.

In previous versions of MuseScore, we did often get complaints that the concert pitch button was too easy to misinterpret.  We tried finessing it with how the border was drawn and other subtleties, but I don't know that any version of MuseScore has ever had anything completely satisfactory.

The switch idea is OK in principle but seems cluttered to me.  I could imagine some variation on that idea working, although frankly, I'd miss the word "concert" in there as it just so universal and it's what I'm looking for.

**Comment by bmarwell** [comment]
I talked to a native British English speaker never having used notation software before. She told me, concert pitch sounds like a tuning thing to her, and she wouldn't have known from the name and symbol. If you ask Google or an AI, sounding pitch would be way less ambiguous. Even if it is not the "industry standard", correct and less ambiguous naming should take preference.

**Comment by SteveBlower** [comment]
As a native British player of a transposing instrument I would normally say, for example, "Concert B' when referring to the untransposed pitch sounding B.
